packages/shared/lib/sdk/sync.ts

```diff
@@ -13,11 +13,20 @@ import type { SyncConfig } from '../models/Sync.js';
 import type { RunnerFlags } from '../services/sync/run.utils.js';
 import { validateData } from './dataValidation.js';
 import { NangoError } from '../utils/error.js';
-import { stringifyAndTruncateLog } from './utils.js';
-import type { DBTeam } from '@nangohq/types';
+import type { DBTeam, MessageRowInsert } from '@nangohq/types';
 
 const logger = getLogger('SDK');
 
+export const oldLevelToNewLevel = {
+    debug: 'debug',
+    info: 'info',
+    warn: 'warn',
+    error: 'error',
+    verbose: 'debug',
+    silly: 'debug',
+    http: 'info'
+} as const;
+
```

Comment:
duplicating this here because shared doesn't depend on `package/logs`. I will remove it from `logs` once it is not used by `persist` anymore